projects/eventonica/README.md

```diff
@@ -15,9 +15,7 @@ You'll work on this project over the next 5 weeks. As you learn more web develop
 1. [Part 4 - API's](./eventonica-part4-apis.md)
 1. [Part 5 - Express Backend](./eventonica-part5-express-backend.md)
 1. [Part 6 - Postgres Database](./eventonica-part6-postgres.md)
-
 1. [Part 7 - React App](./eventonica-part7-react.md)
```

Comment:
Just add a new line after line number 18 `1. [Part 7 - React App](./eventonica-part7-react.md)` @soulhunter10737

Reply:
![image](https://user-images.githubusercontent.com/56026147/76186240-b7ff4e00-61f7-11ea-893d-c33418590e1f.png)

Reply:
I added a new line without <br/> tag as you mentioned earlier.

Reply:
![image](https://user-images.githubusercontent.com/56026147/76186294-e67d2900-61f7-11ea-9337-553eebdfe74b.png)